content/en/blog/2023/submitting-your-first-conference-talk.md

```diff
@@ -0,0 +1,148 @@
+  solve a problem
+- Diving deep on a specific topic or technology
+- Doing a bake-off or round-up of comparable tools
+
+## Where to submit your talk
+
+I am thrilled to see the resurgence in conferences that is currently underway.
+There is no shortage of venues in need of up-and-coming speakers.
+
+As a starting point, I would recommend looking at smaller community-organized
+conferences like DevOpsDays and Code Camps. Often these conferences are hosted
+by a local meetup group, which makes it very easy to get connected with the
+organizers and ask them exactly what kind of talks they are looking for.
+
+Don’t be afraid to submit the same talk to multiple conferences (unless the
+conferences disallow this, but most don’t). Here are some websites I use to find
+conferences:
+
+- [GitHub - scraly/developers-conferences-agenda](https://github.com/scraly/developers-conferences-agenda)
+- [SeeCFP](https://seecfp.com/)
+- [Sessionize](https://sessionize.com/)
+
+## Writing your abstract
+
+Some conferences just require a few sentences or bullet points about your talk,
+while others may require a few pages of information including a detailed
+abstract and bio. In either case, remember to think about the abstract from the
+(potential) audience’s point of view.
+
+It’s important to cover what technologies you will discuss, of course, but you
+also want to make sure you do a proper introduction to the use case for people
+who may not know the technology.
+
+One of the most valuable talks I ever attended was about a topic I needed
+(debugging strategies) in a language I had never used at that point (.NET).
+
+So really, what you want to tell your audience in your abstract is: what will
+they learn from attending your talk?
+
+## Making the most of your conference experience
+
+Making the most of a conference could be a topic for an entirely new blog post,
+but I want to share my top two most important tips:
+
+1. Wear comfortable shoes
+2. Leave room for serendipity (the hallway track)
+
+The first point should be obvious. Most of us are used to spending 10-12 hours a
+day on our butts. Shifting that weight to our feet for a week can be a lot.
+
+The second point is less obvious. At my first conference I felt that I was
+missing out any time I wasn’t in a session. Now I know better. While the talks
+are amazing for revving my mental engine, the “Hallway Track” is where the best
+conversations and connections happen.
+
+The only way to take advantage of this is if you leave some wiggle room in your
+schedule and aren’t afraid to take breaks when you need them.
+
+## Credits
+
+This post was inspired by the discussion in the OTel End User Working Group’s
+session on writing CFPs to help OTel community members submit talks for the
+upcoming conferences, including
+[Observability Day at KubeCon EU](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/cncf-hosted-co-located-events/observability-day/).
```

Comment:
this has been stated in the introduction already, so you might consider removing it?

Reply:
I moved the specific call out to a new paragraph in the "Where to submit your talk session" — we could remove this next week after the deadline.